benchmarks/sparse/dlmc/utils.py

```diff
@@ -39,9 +39,9 @@ def csr_to_coo(indices, indptr, shape):
 
 def load_sparse_matrix(path, device):
     with open(path, 'r') as file:
-        nrows, ncols, nnz = map(lambda el: int(el), file.readline().split(', '))
```

Comment:
how come the original author didn't just write int 🤔

Reply:
Exactly haha